Add files via upload（mike)

mike3104 opened the pull request.

Commit:
index.htmlのレビュー欄の開閉機能(js)　作りました※不完全

・不完全点
★一番上のレビューしか開閉しない。（全部同時に開閉してしまうとおもったら違った）
★そもそもこの開閉方式がよいのか？動作の速さを考えたら、よりよいコードがあるかも。

Commit:
検索窓　1つにしました。

position:rabsolute;により検索窓をひとつにすることができました。
jsにより、クリックで検索枠が出てくるように、コードを記載します。

Commit:
review.htmlとreview.cssです。ご参考にお使いください。

Commit:
デザイン改善、js　レビュー投稿欄　開閉機能の改善

・Qiitaのデザインにより近づけました。また、細かいデザインも改善しました。
・js　レビュー投稿欄　開閉機能の改善をしました。（レビュー欄2個目以降にはやはり開閉機能が働かず、、、）

kakudaisuke reviewed (changes requested):
要件とデザインを見直したので、大変お手数ですがそれを元に再構築お願いします！今日9/8の打ち合わせで説明します。
https://docs.google.com/presentation/d/1rbOul24zd_Xoc3eM_WKgtkZ7btJNBUyaj2nkjGrm2hs/edit?usp=sharing

Commit:
ログインボタンのアコーディオンメニュー実装、各ボタンのホバー機能

Commit:
footerのボタンのhover設定、favicon設定

Commit:
faviconの追加

さきほどfaviconを追加できていなかったので、再コミット。

ghost commented:
なぜかfaviconのアイコンをpushできず。

Commit:
tag.html　tag.cssのたたき台作成

tag.html　tag.cssのたたき台作成を作成しました。
ほぼQiitaを真似て作成していますので、特に問題ないと思いますが、
今の方向性で問題がないか一度ご確認いただけるとありがたいです。

Commit:
ライブハウスのページ（例　BLUE NOTE 東京）のHTMLとCSSのたたき台を作成しました。

ライブハウスのページ（例　BLUE NOTE 東京）のHTMLとCSSのたたき台を作成しました。
作ったところなので、レビューいただき、方向性に間違いないかどうかご確認いただけるとありがたいです。

Commit:
userpage.htm　cssのたたき台lを作成しました。

userpage.html　cssのたたき台を作成しました。
投稿内容の編集削除報告のアコーディオンメニューですが、javascriptが必要なようです。しかも今私が苦戦しているものとおそらく同じようなコードのものかと思われます、、、涙
避けて通してはくれないみたいです。

Commit:
userpage.html 貼り付け忘れ

kakudaisuke commented on userpage.html:
```diff
@@ -0,0 +1,272 @@
+                    <div class="main-userinfo-upperColumn-left">
+                        <div class="main-userinfo-userName">junko666</div>
+                        <div class="main-userinfo-post">
+                            <div class="main-userinfo-postNumberText">投稿記事数:</div>
```
投稿記事数: と132を綺麗に水平に並べられますか？

kakudaisuke commented on userpage.html:
```diff
@@ -0,0 +1,272 @@
+                <div class="main-reviewText">
+                    <div class="main-reviewText-upperColumn">
+                        <div class="main-reviewText-upperColumn-left">
+                            <div class="main-liveDate">2020/8/26 17:30-18:00</div>
```
日時と演奏者を削除して、タイトルとしてください。
タグはdivでなく、hタグの方がいいと思います。

kakudaisuke commented on userpage.html:
```diff
@@ -0,0 +1,272 @@
+
+                            <input id="main-reviewText-acd-check" class="main-reviewText-acd-check" type="checkbox">
+                            <label class="main-reviewText-acd-label" for="main-reviewText-acd-check">
+                                <img src='images/icons8-menu-24.png' class="menuTab"></label>
```
これは意図通りの表示になっていますか？縦長の図になっていますが。

mike3104 replied:
あえてrelativeにして縦長になっています。（これでよいとは思っていません。）
absoluteにすると縦長は解消しますが、acdメニューはすべて同じ位置から出てくることになりますので、見た目は問題ないものの、別の問題が生じます。
今回のコードはabsoluteにしておきます。

kakudaisuke commented on userpage.html:
```diff
@@ -0,0 +1,272 @@
+                            <input id="main-reviewText-acd-check" class="main-reviewText-acd-check" type="checkbox">
+                            <label class="main-reviewText-acd-label" for="main-reviewText-acd-check">
+                                <img src='images/icons8-menu-24.png' class="menuTab"></label>
+                            <div class="main-reviewText-acd-content">
```
このメニューバー一旦開くと閉じなくなるのすが、閉じるようにできますか？
それとここ、<ul>にするといいかもしれませんね。

mike3104 replied:
今回あげたものは閉じると思います。（前回も閉じるのを確認したはずなんですが、）
あとご提案がなぜか真っ白で、ご提案内容がわかりません。

kakudaisuke commented on userpage.html:
```diff
@@ -0,0 +1,272 @@
+            <main>
+                <div class="main-userinfo">
+                    <div class="main-userinfo-upperColumn-left">
+                        <div class="main-userinfo-userName">junko666</div>
```
- もう少しフォントサイズを大きくしてもいいかなと思います。

- ここはdivでなくhタグの見出しが適切かと

kakudaisuke commented on userpage.html:
```diff
@@ -0,0 +1,272 @@
+
+                    <div class="main-reviewText-middleColumn">
+                        <input id="openCloseCheckBox" type="checkbox" />
+                        <div id="main-reviewText">
```
ここはpタグが良いのではないでしょうか？

kakudaisuke commented on tagpage.html:
```diff
@@ -0,0 +1,535 @@
+<!DOCTYPE html>
```
こちらもuserpage.htmlと同様の修正をお願いします！

kakudaisuke commented on tag.html:
```diff
@@ -0,0 +1,633 @@
+        <main>
+            <div class="main-tag-title">
+                <span>タグ一覧</span>
+                <div class="page-number">1ページ目
```
１ページに全て収めるという話をしていたと思うので、その前提でいろいろ修正をお願いします。

kakudaisuke commented on tag.html:
```diff
@@ -0,0 +1,633 @@
+            <div class="main-list">
+                <div class="main-list-row">
+                    <div class="main-list-row-left">
+                        <div class="TagList-item" data-count="#">
```
タグの表示をもう少し大きくしてください。
それと数字は一旦不要です。
ここってimgファイルで表示しますか？

kakudaisuke commented on review.html:
```diff
@@ -0,0 +1,132 @@
+            <div class="right-container">
+                <main>
+                    <div class="main-container">
+                        <div class="main-review-title">レビュー投稿</div>
```
ここはhタグがいいかと思います。

kakudaisuke commented on review.html:
```diff
@@ -0,0 +1,132 @@
+                                <textarea name="review" cols="40" rows="8"></textarea>
+                            </div>
+                            <div class="upload">
+                                <input type="submit" value="投稿する">
```
配置はセンターがいいですね。

kakudaisuke commented on review.html:
```diff
@@ -0,0 +1,132 @@
+                                <input type="submit" value="投稿する">
+                            </div>
+                        </form>
+                    </div>
```
footerとの間にもう少しmarginが欲しいです。

kakudaisuke reviewed (changes requested):
全体的にdivタグを使用しすぎている感じがします。その要素の役割を考えて、hタグ、pタグなどに変更してみてはいかがでしょうか。
それとslackにも書きましたが、divの閉じタグが遠い場合は閉じタグに印をお願いします！
たくさん進めてくれて本当にありがとうございます！

Commit:
いろいろ追記・修正しました。

cssをひとつにまとめた
cssにコメント付与
HTMLの閉じタグにコメント付与
NAVのデザイン修正

Commit:
かくさんのコメントを踏まえて追加修正しました

Commit:
tag.html 微修正

Commit:
font awesomeを反映いたしました。

ご確認ください。

kakudaisuke commented:
@mike3104 だいぶ整ってきて、さらによくなってきましたね！おつかれさまです。マージする前にfront-endブランチとコンフリクトがあるみたいなので、This branch has conflicts that must be resolvedの`Resolve conflicts`で解消してもらえますか？これをしないとマージできないらしいです。少しややこしいと思うので、やり方わからなかったら聞いてください〜

Commit:
Merge branch 'front-end' into mike
